staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -122,10 +122,10 @@ func decodeLiveManagedFields(liveObj runtime.Object) (Managed, error) {
 	return managed, nil
 }
 
-func decodeManagedFields(liveObj, newObj runtime.Object, ignoreManagedFieldsFromRequestObject bool) (Managed, error) {
+func decodeManagedFields(liveObj, newObj runtime.Object, subresource string) (Managed, error) {
```

Comment:
If this doesn't need to know the specific subresource used, I would keep the flag with the proper name as it gives more information about why this is used for.

Reply:
Good point, let me change it back.